src/site/content/en/blog/sms-otp-form/index.md

```diff
@@ -0,0 +1,275 @@
+
+Some websites use `type="tel"` for OTP input fields since it also 
+turns the mobile keyboard to numbers only (including `*` and `#`) when
+focused. This hack was used in the past when `inputmode="numeric"`
+wasn't widely supported. Since [Firefox started supporting
+`inputmode="numeric"`](https://github.com/mdn/browser-compat-data/pull/6782),
+there's no need to use the semantically incorrect `type="tel"` hack.
+
+### `autocomplete="one-time-code"`
+
+[`autocomplete`](https://developer.mozilla.org/docs/Web/HTML/Attributes/autocomplete) 
+attribute lets developers specify what permission the browser 
+has to provide autocomplete assistance and informs the browser about the 
+type of information expected in the field.
+
+With `autocomplete="one-time-code"` whenever a user receives an SMS message while a 
+form is open, the operating system will parse the OTP in the SMS heuristically and
+the keyboard will suggest the OTP for the user to enter. It works only on Safari 12 and 
+later on iOS, iPadOS, and macOS, but we strongly recommend using it, because it is an 
+easy way to improve the SMS OTP experience on those platforms.
+
+<figure class="w-figure" style="width:300px; margin:auto;">
+  <video controls autoplay loop muted class="w-screenshot">
+    <source src="https://storage.googleapis.com/web-dev-assets/sms-otp-form/ios-safari.webm" type="video/webm">
+    <source src="https://storage.googleapis.com/web-dev-assets/sms-otp-form/ios-safari.mp4" type="video/mp4">
+  </video>
+  <figcaption class="w-figcaption">
+    `autocomplete="one-time-code"` in action.
+  </figcaption>
+</figure>
+
+`autocomplete="one-time-code"` improves the user experience, but there's more you
+can do by [ensuring that the SMS message complies with the origin-bound message
+format](#format).
+
+{% Aside %}
+Optional attributes:
+
+* [`pattern`](https://developer.mozilla.org/docs/Web/HTML/Attributes/pattern)
+  specifies the format that the entered OTP must match. Use regular expressions to specify
+  the matching pattern, for example, `\d{6}` constrains the OTP to a six digit string. Learn more about the 
+  `pattern` attribute in [Use JavaScript for more complex real-time validation] 
+  (https://developers.google.com/web/fundamentals/design-and-ux/input/forms#use_javascript_for_more_complex_real-time_validation)
+  
+* [`required`](https://developer.mozilla.org/docs/Web/HTML/Attributes/required)
+  indicates that a field is required.
+
+For more general form best practices, [Sam Dutton](/authors/samdutton/)'s
+[Sign-in form best practices](/sign-in-form-best-practices/) is a great starting
+point.
+{% endAside %}
+
+## Format the SMS text {: #format }
+
+Enhance the user experience of entering an OTP by aligning with 
+[the origin-bound one-time codes delivered via SMS](https://wicg.github.io/sms-one-time-codes/) 
+specification. 
+
+The format rule is simple: Finish the SMS message with the receiver domain
```

Comment:
@mihajlija Regarding this change, I added `receiver` here to clarify that the domain should be bound to the receiver domain rather than the sender domain in case they are different. If you have any alternative suggestion, please let me know.